packages/mux-player/src/template.ts

```diff
@@ -62,14 +63,25 @@ export const content = (props: MuxTemplateProps) => html`
         ? getPosterURLFromPlaybackId(props.playbackId, props.thumbnailTime ?? props.startTime, props.tokens.thumbnail)
         : false}"
     >
-      ${props.playbackId && props.streamType !== StreamTypes.LIVE && props.streamType !== StreamTypes.LL_LIVE
-        ? html`<track
-            label="thumbnails"
-            default
-            kind="metadata"
-            src="${getStoryboardURLFromPlaybackId(props.playbackId, props.tokens.storyboard)}"
-          />`
-        : html``}
+      <video
```

Comment:
blocking: How would you feel about CustomVideoElement just baking in using `is="castable-video"` per the discussion w/Heff on the `<media-chrome/>` side? Feels lower risk in terms of refactors and makes ChromeCast support also easier for folks using `<mux-video/>` + `<media-chrome/>` (per e.g. our TMI presentations).

Reply:
I'm fine with that but to me the current solution feels a bit easier than adding castable-video and castable-audio in each of 4 the Mux elements WC and React...

Maybe we could add Casting support later to those elements with this solution first

Reply:
ah I just meant literally updating the template of CustomVideoElement from:

```html
<slot name="media"><video part="video" crossorigin></video></slot>
```

to:
```html
<slot name="media"><video part="video" is="castable-video" crossorigin></video></slot>
```
(and importing it into the module)

Reply:
def cool with holding off as a followup if you think this is the path of lower risk